cms/templates/widgets/sock.html

```diff
@@ -69,28 +37,22 @@ <h3 class="title">${_("Request help with {studio_name}").format(studio_name=sett
             'text': _('edX Partner Portal'),
             'condition': is_edx_domain
         }, {
+            'href': 'https://stanfordonline.zendesk.com/hc',
+            'sr_mouseover_text': _('Access Course Staff Support on the Help Center to submit or review support tickets'),
+            'text': _('Help Center'),
+            'condition': True
+        }, {
             'href': 'https://open.edx.org',
             'sr_mouseover_text': _('Access the Open edX Portal'),
             'text': _('Open edX Portal'),
             'condition': not is_edx_domain
         }, {
-            'href': 'https://www.edx.org/course/overview-creating-edx-course-edx-edx101#.VO4eaLPF-n1',
-            'sr_mouseover_text': _('Enroll in edX101: Overview of Creating an edX Course'),
-            'text': _('Enroll in edX101'),
-            'condition': True
-        }, {
-            'href': 'https://www.edx.org/course/creating-course-edx-studio-edx-studiox',
```

Comment:
This sends people to edX and they'll have to create an account there to be able to take the course.
We can keep them, but I was just removing clutter in the list of buttons/links.